assignment-02.Rmd

```diff
@@ -17,12 +17,12 @@ you assignment as a knitted `.pdf` (prefered) or `.html` file.*
     b.  Assign the result of `10/3` to the variable `b`. Display `b`.
         (0.25 marks)
 
-    c.  Assign the product of `a` and `b` to `c`. Display `c`. (0.25
+    c.  Assign the product of `a` and `b` to `product`. Display `product`. (0.25
         marks)
 
     d.  Write a function that adds two numbers and returns their sum.
-        Use it to assign the sum of `a` and `b` to `c`. Display `c`. (In
-        practice, there is already a more sophisticated built-in
+        Use it to assign the sum of `a` and `b` to `product`. Display `product`.
```

Comment:
But the sum of two things is not a product... Maybe name it `result` instead?

Reply:
*head meets desk*